src/styles/App.css

```diff
@@ -1 +1,322 @@
-/* Styles */
+body {
```

Comment:
## Congratulations

Reply:
@OLIPLICHE Hi, thank you for your feedback and for approving my project as well! Have a great day.